sycl/source/detail/usm/usm_impl.cpp

```diff
@@ -232,240 +231,319 @@ void free(void *Ptr, const context &Ctxt, const detail::code_location &CL) {
+void *
+aligned_alloc_shared(size_t Alignment, size_t Size, const device &Dev,
+                     const context &Ctxt,
+                     const property_list &PropList _CODELOCPARAMDEF(&CL)) {
+  _CODELOCARG(&CL);
   return detail::usm::alignedAlloc(Alignment, Size, Ctxt, Dev, alloc::shared,
-                                   CL, PropList);
+                                   PropList, CL);
 }
 
-void *aligned_alloc_shared(size_t Alignment, size_t Size, const queue &Q,
-                           const detail::code_location CL) {
-  return aligned_alloc_shared(Alignment, Size, Q.get_device(), Q.get_context(),
-                              CL);
+void *aligned_alloc_shared(size_t Alignment, size_t Size,
+                           const queue &Q _CODELOCPARAMDEF(&CL)) {
+  _CODELOCARG(&CL);
+  return detail::usm::alignedAlloc(Alignment, Size, Q.get_context(),
+                                   Q.get_device(), alloc::shared,
+                                   property_list{}, CL);
 }
 
-void *aligned_alloc_shared(size_t Alignment, size_t Size, const queue &Q,
-                           const property_list &PropList,
-                           const detail::code_location CL) {
-  return aligned_alloc_shared(Alignment, Size, Q.get_device(), Q.get_context(),
-                              PropList, CL);
+void *
+aligned_alloc_shared(size_t Alignment, size_t Size, const queue &Q,
+                     const property_list &PropList _CODELOCPARAMDEF(&CL)) {
+  _CODELOCARG(&CL);
+  return detail::usm::alignedAlloc(Alignment, Size, Q.get_context(),
+                                   Q.get_device(), alloc::shared, PropList, CL);
 }
 
 // single form
 
 void *malloc(size_t Size, const device &Dev, const context &Ctxt, alloc Kind,
-             const detail::code_location CL) {
+             const property_list &PropList _CODELOCPARAMDEF(&CL)) {
+  _CODELOCARG(&CL);
   void *RetVal = nullptr;
 
   if (Kind == alloc::host) {
-    RetVal = detail::usm::alignedAllocHost(0, Size, Ctxt, Kind, CL);
+    RetVal = detail::usm::alignedAllocHost(0, Size, Ctxt, Kind, PropList, CL);
   } else {
-    RetVal = detail::usm::alignedAlloc(0, Size, Ctxt, Dev, Kind, CL);
+    RetVal = detail::usm::alignedAlloc(0, Size, Ctxt, Dev, Kind, PropList, CL);
   }
 
   return RetVal;
 }
 
-void *malloc(size_t Size, const device &Dev, const context &Ctxt, alloc Kind,
-             const property_list &, const detail::code_location CL) {
-  return malloc(Size, Dev, Ctxt, Kind, CL);
+void *malloc(size_t Size, const device &Dev, const context &Ctxt,
+             alloc Kind _CODELOCPARAMDEF(&CL)) {
+  _CODELOCARG(&CL);
+  void *RetVal = nullptr;
+
+  if (Kind == alloc::host) {
+    RetVal =
+        detail::usm::alignedAllocHost(0, Size, Ctxt, Kind, property_list{}, CL);
+  } else {
+    RetVal = detail::usm::alignedAlloc(0, Size, Ctxt, Dev, Kind,
+                                       property_list{}, CL);
+  }
+
+  return RetVal;
```

Comment:
```suggestion
  if (Kind == alloc::host)
    return detail::usm::alignedAllocHost(0, Size, Ctxt, Kind, property_list{},
                                         CL);
  return detail::usm::alignedAlloc(0, Size, Ctxt, Dev, Kind, property_list{},
                                   CL);
```
Same applies to the next couple functions.

Reply:
Note: There are more functions like this. I only made the suggestion for the first one, but the changes should be applied to all of them.